packages/rsocket-ws-gateway/src/createGatewayProxy.ts

```diff
@@ -9,9 +9,24 @@ interface Proxy {
   [state: string]: any;
 }
 
-export function createGatewayProxy(url: string, definition: MicroserviceApi.ServiceDefinition): Promise<Proxy>;
-export function createGatewayProxy(url: string, definition: MicroserviceApi.ServiceDefinition[]): Promise<Proxy[]>;
-export function createGatewayProxy(url: string, definitions: any): any {
+export function createGatewayProxy(
+  url: string,
+  definition: MicroserviceApi.ServiceDefinition,
+  requestResponse?: any,
+  requestStream?: any
+): Promise<Proxy>;
+export function createGatewayProxy(
+  url: string,
+  definition: MicroserviceApi.ServiceDefinition[],
+  requestResponse?: any,
+  requestStream?: any
```

Comment:
add API for requestStream
```suggestion
  requestStream?: RequestStream
```